### ISSUE TITLE
Add ability to upload an image to NMDC submissions via submission portal

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/nmdc-server#1608]
/study page of the submission portal should have a "upload image" spot to add a photo of the PI.

![Screenshot 2024-08-15 at 12 44 44 PM](https://github.com/user-attachments/assets/90f99176-4f80-429e-9d7a-35d47b1b6862)

- shrink the PI ORCID box
- add a upload image box
  - How do we store these? Where should it go?
- info grey text should read "An image of the PI for a [research study](https://data.microbiomedata.org/details/study/nmdc:sty-11-5tgfr349) or a logo for a [consortia](https://data.microbiomedata.org/details/study/nmdc:sty-11-nxrz9m96) to share on the study landing page.

**Comment by pkalita-lbl** [comment]
A few miscellaneous thoughts:
 
1. I think we have a path forward on the technical details of storing and retrieving media. See #1612 for details. 
2. The annotation in the screenshot says “A image of the PI for a research study or logo for a consortia”. Those two types of images go to different places in the schema (`Study.principal_investigator.profile_image_url` and  `Study.study_image.url` respectively). So in order to direct the image to the correct place, we’d need to add something to the Submission Portal UI that asks the user whether the study is a research study or a consortia. Not a big deal, but we’d need to decide where that would go and what the language around it would be.
3. Part of the Data Portal ingest process currently tries to avoid storing duplicate image data. Say we already have a study with a PI named “Morgan Jones”. If a second study comes in with a PI named “Morgan Jones”, the ingest will always reuse the existing PI image (provided by the first study) no matter what else the second study contains.
    1. This behavior is slightly questionable because matching by name is a weird choice when we have the PI email. Plus, the PI name isn’t even technically required in the Submission Portal.
    2. If we keep this behavior while allowing user image uploads, this could be confusing to users. If they have one submission accepted and ingested, then come back later and make a second submission with a different PI image, they might rightfully wonder why that second image they uploaded isn’t displayed on the data portal after ingest.
    3. There is something to be said, though, for keeping PI images consistent across studies. It would be nice if one PI creates a number of submissions that they don’t have to keep uploading the same photo over and over. That gets into the idea of attaching an image to the user's portal profile (i.e. the `user_logins` table in Postgres). That might simplify some use cases, but it also might complicate the cases where the user creating the submission is _not_ the PI.

**Comment by mslarae13** [comment]
>we’d need to add something to the Submission Portal UI that asks the user whether the study is a research study or a consortia. Not a big deal, but we’d need to decide where that would go and what the language around it would be.
😬 -- we decided not to do that so we don't confuse people..

Maybe we have a button or a drop down option, PI Photo or Project logo? IDK, then people might just always do logo... curious on @lamccue 's thoughts

>This behavior is slightly questionable because matching by name is a weird choice when we have the PI email. Plus, the PI name isn’t even technically required in the Submission Portal.
Agreed, it should be ORCID

>f we keep this behavior while allowing user image uploads, this could be confusing to users. If they have one submission accepted and ingested, then come back later and make a second submission with a different PI image, they might rightfully wonder why that second image they uploaded isn’t displayed on the data portal after ingest.

>There is something to be said, though, for keeping PI images consistent across studies. It would be nice if one PI creates a number of submissions that they don’t have to keep uploading the same photo over and over. That gets into the idea of attaching an image to the user's portal profile (i.e. the user_logins table in Postgres). That might simplify some use cases, but it also might complicate the cases where the user creating the submission is not the PI.
Hmm.. you're making me think about a bigger question here... We have this desire of "multiple submissions of metadata for 1 study"... this sounds similar.. 1 PI, multiple studies, Multiple submissions.
A good thing to noodle on.

**Comment by mslarae13** [comment]
@pkalita-lbl is it safe to say this is no longer blocked, but in progress?

**Comment by pkalita-lbl** [comment]
Yeah, it's in progress. It got put on the back burner a bit in the last week or two. We need to circle back on some requirements.

**Comment by lamccue** [comment]
> > we’d need to add something to the Submission Portal UI that asks the user whether the study is a research study or a consortia. Not a big deal, but we’d need to decide where that would go and what the language around it would be.
> 
> 😬 -- we decided not to do that so we don't confuse people..
> 
> Maybe we have a button or a drop down option, PI Photo or Project logo? IDK, then people might just always do logo... curious on [@lamccue](https://github.com/lamccue) 's thoughts
> 

I'm thinking that consortia are the unusual case. For all of the consortia that we have so far, we took the lead to add those, usually working with someone from each consortium.  So I would think that the vast majority (since I hate absolutes) of submissions to the submission portal will be research studies. Which means, I don't think we need to ask if it is a research study or a consortium.  Instead, just assume everything is a research study; we could maybe have an information icon that could pop up to say that if the submission is a consortium, to contact us for help (since I do expect these to be the rare edge case).

If we did that, then with regard to the image, that would mean that we always expect a PI Photo. Unless we want to loosen that constraint and allow the option for a logo. We used logos for the big parent projects like the BRCs. I'd be supportive of a little flexibility in letting someone chose a photo or a logo.

**Comment by mslarae13** [comment]
Tested in -dev and from the user side looks good! No testing in -dev for ingest. I made some comments in https://github.com/microbiomedata/nmdc-server/issues/1788 but nothing that needs to be addressed ASAP (& should probably be split into sub-issues if these will be addressed.
